jerry-core/ecma/operations/ecma-conversion.c

```diff
@@ -278,77 +278,9 @@ ecma_op_to_boolean (ecma_value_t value) /**< ecma value */
  */
 ecma_value_t
```

Comment:
One more thing:

This should be an `inline` function with `JERRY_ATTR_ALWAYS_INLINE` attribute.